inscriptions/ISic3076.xml

```diff
@@ -181,17 +186,19 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI331998">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Σηιανέ,<lb n="2"/>μνησθῇ<lb n="3"/>σου ὁ θε-<lb n="4"/> ός.
-                    <lb n="5"/>
+                    <lb n="1"/>Σηιανέ
+                    <lb n="2"/>μνησθῇ <supplied reason="lost">ὦ</supplied> <supplied reason="lost">θεὸς</supplied> <supplied reason="lost">τῷ</supplied> <supplied reason="lost">δούλῳ</supplied>
+                    <lb n="3" break="no"/>σοῦ Ἀθε<gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
Is there a word break here? Line 2 seems to end in δούλῳ.

Reply:
Actually, I would follow Griesheimer here, who reasonably seems to think that this text is in fact complete, with a fourth line:
![image](https://user-images.githubusercontent.com/25080386/104006007-d2021800-519d-11eb-9464-829fb3b6e18c.png)